addons/web/static/src/legacy/js/core/abstract_service.js

```diff
@@ -76,7 +76,7 @@ var AbstractService = Class.extend(Mixins.EventDispatcherMixin, ServicesMixin, {
                 }
             }
         }
-        serviceRegistry.onAdd((serviceName, Service) => {
+        core.serviceRegistry.onAdd((serviceName, Service) => {
```

Comment:
the diff in this file is no longer necessary, right?

Reply:
I believe it is.
In tests we do `legacy.core.serviceRegistry = legacyParams.serviceRegistry;`
so the abstract service code should not hold on serviceRegistry in its closure (because it won't be modified by tests ), rather, if it holds on core, we can access any of its properties dynamically

To avoid this this, we can, in tests, override serviceRegistry.map instead

Reply:
trickyyyyy